src/components/Orders/Orders.js

```diff
@@ -71,17 +70,26 @@ Orders.propTypes = {
   rowMapping: PropTypes.objectOf(PropTypes.shape(DATA_MAPPING)),
   className: PropTypes.string,
   cancelOrder: PropTypes.func.isRequired,
-  parentWidth: PropTypes.number,
+  /**
+   * If true, show the Orders in a condensed mobile layout. By default
+   * the mobile layout will be enabled when the screen size is below 576.
```

Comment:
just thinking if we change BREAKPOINT_SMALL in future, may forgot to update 576 in prop-desc can we use BREAKPOINTS.SM in prop-desc line above ?